### ISSUE TITLE
Appearance and article view improvements

### ISSUE THREAD

**Issue by bb1a2** [mortalis13/HomeDictionary#4]
- CSS support for customization
- dictionary icon
- put `(*)->(*)` at the top of the language list.
- use tabs or something at the top (below search) to switch groups of dictionaries: All, mygroup1, mygroup2, etc.
- jump to dict on page (like HTML # anchors)
- support media (images, audio)

**Comment by mortalis13** [comment]
"Jump to dict on page"?
You mean select (scroll to) a spesific dictionary translation, with some kind of contents?

No media sorry, only text.

**Comment by bb1a2** [comment]
> "Jump to dict on page"? You mean select (scroll to) a spesific dictionary translation, with some kind of contents?

Yes. Instead of having to scroll manually if you have many dictionaries. Like a Table of Contents.

> No media sorry, only text.

That's disappointing. Is it possible to know why? Thanks
